### PR TITLE
kyma-cli: add DefaultKymaVersion

### PR DESCRIPTION
**Pull request by chenrui333**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

followup of #66039

### CODE REVIEW

File: Formula/kyma-cli.rb [hunk 1]
```diff
@@ -19,6 +20,8 @@ def install
     ldflags = %W[
       -s -w
       -X github.com/kyma-project/cli/cmd/kyma/version.Version=#{version}
+      -X github.com/kyma-project/cli/cmd/kyma/install.DefaultKymaVersion=master
```
Comment (suleymanakbas91) [inline Formula/kyma-cli.rb:23]:
```suggestion
      -X github.com/kyma-project/cli/cmd/kyma/install.DefaultKymaVersion=#{version}
```

File: Formula/kyma-cli.rb [hunk 2]
```diff
@@ -19,6 +20,8 @@ def install
...
+      -X github.com/kyma-project/cli/cmd/kyma/upgrade.DefaultKymaVersion=master
```
Comment (suleymanakbas91) [inline Formula/kyma-cli.rb:24]:
```suggestion
      -X github.com/kyma-project/cli/cmd/kyma/upgrade.DefaultKymaVersion=#{version}
```

File: Formula/kyma-cli.rb (hunk 1 above)
Comment (suleymanakbas91) [reply Formula/kyma-cli.rb:23]: Master is for the binaries created from the master branch. For the releases, we're using the release version, which is the same as the `version` here.

File: Formula/kyma-cli.rb (hunk 2 above)
Comment (suleymanakbas91) [reply Formula/kyma-cli.rb:24]: Same as above.

**Review by suleymanakbas91 — CHANGES_REQUESTED** [review]

File: Formula/kyma-cli.rb (hunk 1 above)
Comment (chenrui333) [reply Formula/kyma-cli.rb]: Sounds good, thanks for the catch! :)

**Review by SMillerDev — APPROVED** [review]

**Review by suleymanakbas91 — APPROVED** [review]

**Pull request closed without merging**